android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/GpsFactory.java

```diff
@@ -217,14 +226,19 @@ public void attachLayout(Context context, @NonNull JSONObject jsonObject, @NonNu
             }
         }
 
-        latitudeTv.setText(String.format(context.getString(R.string.latitude), latitude));
-        longitudeTv.setText(String.format(context.getString(R.string.longitude), longitude));
-        altitudeTv.setText(String.format(context.getString(R.string.altitude), altitude));
-        accuracyTv.setText(String.format(context.getString(R.string.accuracy), accuracy));
+        latitudeTv.setText(getText(context, latitude, R.string.latitude));
+        longitudeTv.setText(getText(context, longitude, R.string.longitude));
+        altitudeTv.setText(getText(context, altitude, R.string.altitude));
+        accuracyTv.setText(getText(context, accuracy, R.string.accuracy));
 
         dataView.setTag(R.id.raw_value, constructString(latitude, longitude));
     }
 
+    @NotNull
+    public String getText(Context context, String latitude, int p) {
```

Comment:
Perhaps a better param name would b _coordinate_? since it takes both _longitudes_ and _latitude_